ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -13,7 +14,10 @@ import { getTokens } from '../../ducks/metamask/metamask';
 import ZENDESK_URLS from '../../helpers/constants/zendesk-url';
 import { isEqualCaseInsensitive } from '../../../shared/modules/string-utils';
 import { getSuggestedAssets } from '../../selectors';
```

Comment:
Presumably you'll have to repoint this selector since this state will be different as it no longer lives on the TokensController?

Reply:
Good catch, we have a new selector that uses approvals filtered by type. It'll definitely need to be used here.